geopyspark/geotrellis/layer.py

```diff
@@ -440,9 +440,6 @@ def read(cls,
 
         layer_type = LayerType(layer_type)
 
-        if layer_type == LayerType.SPACETIME:
```

Comment:
Sorry, I was being stupid and misread this error message and I thought it could be removed. We actually do want this check, though. Could you please add these two checks back in?